packages/@lightningjs/ui-components-test-utils/src/lightning-test-utils.d.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import lng from '@lightningjs/core';
+import { jest } from '@jest/globals';
+import type {
+  default as TestRenderer,
+  testRenderer
+} from './lightning-test-renderer.d.ts';
+
+export function nextTick(wait?: number): Promise<void>;
+
+export function fastForward(elements: lng.Element[]): void;
+
+type makeCreateComponentConfig = Record<string, any>;
```

Comment:
can this be `Record<string, unknown>` or does it need to be `any`?

Reply:
Ya, switching it to `unknown` should be fine.